### PR TITLE
ft_watcher: add evm token router parser

### PR DESCRIPTION
**Pull request by bingyuyap**
- normalize fast transfer schema into multiple tables. Data can be too sparse when subsequent phases are not executed/observed yet
- upgrade TS version from from v4 to v5 and add Fast Transfer evm sdk
- add contract address for `ArbitrumSepolia`. Other EVM Testnet contracts will be added in the next PR
- add Token Router Parser to parse `placeFastMarketOrder`
- add EVM Fast Transfer Watcher to monitor and persist Fast Market Orders 
- add tests for EVM Fast Transfer Watcher
- update tests for SOL Fast Transfer Watcher

### CODE REVIEW

File: watcher/src/fastTransfer/tokenRouter/parser.ts [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+          'hex'
+        )}/${fastMessage.sequence}`;
+
+        const blockTime = await this.fetchBlockTime(blocks, res.blockNumber);
+
+        return {
+          fast_vaa_id: vaaId,
+          amount_in: fastOrder.amountIn,
+          min_amount_out: fastOrder.minAmountOut,
+          src_chain: toChainId(this.chain),
```
Comment (panoel) [inline watcher/src/fastTransfer/tokenRouter/parser.ts:107]: chainToChainId() would be better here.

File: watcher/tsconfig.json [hunk 2]
```diff
@@ -2,13 +2,19 @@
...
+      ],
       "@wormhole-foundation/example-liquidity-layer-solana/*": [
-        "../node_modules/@wormhole-foundation/example-liquidity-layer-solana/dist/esm/*"
+        "../node_modules/@wormhole-foundation/example-liquidity-layer-solana/dist/cjs/*"
       ]
     }
   },
-  "include": ["scripts", "src", "src/abi/*.json", "src/idls/*.json"]
+  "include": ["scripts", "src", "src/abi/*.json", "src/idls/*.json"],
+  "exclude": ["sdk"]
```
Comment (panoel) [inline watcher/tsconfig.json:19]: Do you need exclude if you have include?

File: watcher/src/watchers/FTEVMWatcher.ts [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+import { FAST_TRANSFER_CONTRACTS, FTChains } from '../fastTransfer/consts';
+import { ethers } from 'ethers';
+import { RPCS_BY_CHAIN } from '../consts';
+import { makeBlockKey } from '../databases/utils';
+import TokenRouterParser from '../fastTransfer/tokenRouter/parser';
+import { MarketOrder } from '../fastTransfer/types';
+
+export type BlockTag = 'finalized' | 'safe' | 'latest';
+
+export class FTWatcher extends Watcher {
```
Comment (panoel) [inline watcher/src/watchers/FTEVMWatcher.ts:14]: This is only used in testing now?

**Review by panoel — DISMISSED** [review]

File: watcher/tsconfig.json (hunk 2 above)
Comment (bingyuyap) [reply watcher/tsconfig.json:19]: Nope, good call. Removed.

File: watcher/src/fastTransfer/tokenRouter/parser.ts (hunk 1 above)
Comment (bingyuyap) [reply watcher/src/fastTransfer/tokenRouter/parser.ts:107]: Changed.

File: watcher/src/watchers/FTEVMWatcher.ts (hunk 3 above)
Comment (bingyuyap) [reply watcher/src/watchers/FTEVMWatcher.ts:14]: Yep! Will have a separate PR to invoke both FTEVMWatcher and FTSolanaWatcher for the actual monitoring

**Review by panoel — APPROVED** [review]

**Pull request merged**